docs/README.adoc

```diff
@@ -122,6 +122,87 @@ Drawings from QCAD
  - Export the  image as 480 x 480 Resolution Auto .png
 
 
+Dealing with translations
+-------------------------
+
+Translation of our documentation is in transition at the moment.
```

Comment:
Added this because I can't tell how many time in 20 years I've seen "at the moment" in FOSS softwares docs where it was at least unclear when that moment actually was, and at worst clearly a looong time ago.

Reply:
Then we have "git blame" - nah, admittedly awkward but just take it as a placeholder that something needs to be done.